test/invoice_item_test.rb

```diff
@@ -28,4 +37,20 @@ def test_invoice_item_has_a_unit_price
     assert_equal 136.35, invoice_item.unit_price
   end
 
+  def test_find_invoice_returns_an_invoice_associated_with_invoice_item
+    result = @invoice_item_repository.invoice_items[9].invoice
+    assert_equal 3, result.customer_id
+  end
+
+  def test_find_item_returns_an_item_associated_with_invoice_item
+    result = @invoice_item_repository.invoice_items[7].item
+    assert_equal 8, result.id
+  end
+
+  private
+
+  def fixtures_directory
+    File.expand_path('../../data/fixtures', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.